cmd/notation/cert.go

```diff
@@ -419,6 +427,16 @@ func removeCertsCore(storeType, namedStore, cert string, all bool, errorSlice []
 	} else {
 		path, err := dir.Path.UserConfigFS.GetPath(dir.TrustStoreDir, "x509", storeType, namedStore, cert)
 		if err == nil {
+			prompt := fmt.Sprintf("Are you sure you want to delete: %q?", path)
+			confirmed, err := ioutil.AskForConfirmation(os.Stdin, prompt, confirmed)
+			if err != nil {
+				errorSlice = append(errorSlice, fmt.Errorf("%s with error \"%s\"", path, err.Error()))
+				return errorSlice
+			}
+			if !confirmed {
+				return errorSlice
+			}
+
```

Comment:
I suggest separate `removeCertsCore` into two functions, e.g.: `removeSomeCerts` and `removeAllCerts`

Reply:
> I suggest split`removeCertsCore` into two functions, e.g.: `removeSomeCerts` and `removeAllCerts`

Good idea. Renamed to removeAllCerts and removeCert.